spec/message.abnf

```diff
@@ -28,6 +28,8 @@ annotation = (function *(s option))
            / reserved-annotation
 
 literal = quoted / unquoted
+quoted = "|" *(quoted-char / quoted-escape) "|"
```

Comment:
Let's start at least some cleanup. Keep the `=` aligned with the other items in this block (yes, I see that `option` is not aligned... yet)

Reply:
Could we take care of alignment in a single pass later, kinda like we resolved to not need Prettier to be automatically styling everything? Here, for instance, we may well later rename this rule as `quoted-literal` given that we also have `quoted-pattern`.

Reply:
🤷 

That's fine, but I'd like to start locking the ABNF down to some degree.